### ISSUE TITLE
NdSolve.java contains carriage return (0x0d) characters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cutelyaware/magiccube4d#100]
```
NdSolve.java has \r characters at the end of every line.  Just in case this
isn't a trivial fix in Windows I have posted a patch here:

http://noh.ucsd.edu/~bmenrigh/ndcr.diff


```

Original issue reported on code.google.com by `bmenr...@gmail.com` on 28 Nov 2009 at 9:47

**Comment by SharkRetriever** [comment]
Not quite sure about this one.
Can be easily fixed in Windows using Notepad++, but isn't "\r\n" the default line break String on Windows?

**Comment by donhatch** [comment]
Seems to have been fixed by 8516f0b63cbed5c5ba0efed4e03e473e352cd321 on 2016/1/15.
